### DISCUSSION TITLE
How do I manage multiple projects using single database?

### DISCUSSION THREAD

**Discussion by pankaj9296** [prisma/prisma#6296] [Q&A]
So I have a big project that has different applications like a Dashboard, an API, and a Homepage. All these are in a separate repo and connected to a single database.

How should I enable them to use the same database?
Considering I've to manage Schema in each project and keep them in sync, also how to I manage migrations, running migrations from all three projects on deployment might cause conflicts.

**Comment by ryands17** [comment]
@pankaj9296 👋 

In this case, you can create an NPM package with your `schema.prisma` and keep this in a separate repo. You can then manage your migrations, models all from this repo and then deploy an NPM package that will be used by other repos so that there is no conflict and all use the same version.

**Reply by tarazena** [reply]
I think Introspecting the DB on each env during build time can work

**Reply by pankaj9296** [reply]
Yeah, creating an NPM package and managing it separately is too much for me.
I would rather go with running migrations from one app and introspecting on another app on build time.

Thanks guys.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
